bin/node/cli/res/dusty.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "Dusty",
-  "id": "dusty4",
+  "id": "dusty5",
```

Comment:
@akru  should we bump the version now? Or will we keep the chain id and manually bump it in a later PR.

Reply:
It’s ok to create new chain_id for Dusty relaunch.